tests/functional/pages/surveys/question.page.js

```diff
@@ -33,6 +33,11 @@ class QuestionPage {
     return browser.click('a[id="top-previous"]')
   }
 
+  next() {
```

Comment:
The reason that was added was because there were issues where the page it was submitting wasn't the one it thought it was. hence we got the errors in travis. i.e.. it would click the submit multiple times on a page and only change page once.